File: TroopersApp/app/src/main/java/com/example/teffe/troopersapp/model/Trooper.java
package com.example.teffe.troopersapp.model;

import java.io.Serializable;

/**
 * Created by teffe on 18/11/2017.
 */

public class Trooper implements Serializable {
    private int id;
    private String name;
    private String imageUrl;
    private String description;
    private Affiliation affiliation;

    public Trooper(int id, String name, String imageUrl, String description, Affiliation affiliation) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.imageUrl = imageUrl;
        this.affiliation = affiliation;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public Affiliation getAffiliation() {
        return affiliation;
    }
}

File: TroopersApp/app/src/main/java/com/example/teffe/troopersapp/model/Trooper.kt
package com.example.teffe.troopersapp.model

import java.io.Serializable

/**
 * Created by teffe on 19/11/2017.
 */

class Trooper(var id: Int,
              var name: String,
              var imageUrl: String,
              var description: String,
              var affiliation: Affiliation)
    : Serializable {
    var isFavorite: Boolean = false
}